### PR TITLE
Improve code quality and create components

### PR DESCRIPTION
**Pull request by tkdeshan**

### CODE REVIEW

File: src/pages/login.js [hunk 1]
```diff
@@ -4,77 +4,75 @@ import users from "../users.json";
 import { useAuth } from "../AuthContext";
 import bcrypt from "bcryptjs";
 import "../assets/styles/main.css";
+import logo from "../assets/images/logo.png";
 
 const LoginForm = () => {
-  const [username, setUsername] = useState("");
-  const [password, setPassword] = useState("");
+  const [enterUsername, setEnterUsername] = useState("");
```
Comment (pankajan05) [inline src/pages/login.js]: @tkdeshan you don't need to provide "enter" in the variable name.

Comment (tkdeshan) [inline src/pages/login.js]: Okay

**Review by tkdeshan — COMMENTED** [review]
resolved

**Review by pankajan05 — APPROVED** [review]
checked

**Pull request merged**